### DISCUSSION TITLE
Multiple Instances on One Machine

### DISCUSSION THREAD

**Discussion by AlanRHurley3372** [moriturus/ktra#48] [Q&A] [answered]
I was wondering if anyone has experience with, it or if it is possible to run two instances of Ktra on different ports of one machine.  Looking for a solution in which one instance is public-facing and credentialed and the other is a private uncredentialed instance. Essentially trying to set up two instances of ktra pointed to the same database. Any help is greatly appreciated.

**Comment by gagbo** [comment, marked as answer]
Hello,

I never tried it, but given the few bugs around the asynchronous I/O operations that ktra does, I would not feel confident running 2 instances targeting the same zone of the filesystem for now.

**Reply by AlanRHurley3372** [reply]
Thank you for the explanation! That helps clear it up a lot.
